tools/ou3_interval_ad.py

```
@@ -0,0 +1,344 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+import math
+from typing import Iterable, Sequence
+
+from ou3_interval import Interval, hull
+import ou3_p5_deployed_quaternion_cayley_cell as QCOMP
+import ou3_p4_group_algebra as GROUP
+
+
+def I(x: float) -> Interval:
```

Ruff (0.16.3)

[error] Ambiguous function name: `I`

(E743)